emcc.py

```diff
@@ -1258,6 +1258,9 @@ def get_last_setting_change(setting):
 
     shared.verify_settings()
 
+    if not shared.Settings.WASM_BACKEND:
+      diagnostics.warning('fastcomp', 'the fastomp compiler will soon be deprecated.  Please switch to the upstream llvm backend as soon as possible and open issues if you have trouble doing so')
```

Comment:
Can we say that it *is* deprecated and will soon be removed? or soon stop being updated?

Reply:
maybe saying something about when it's removed/no longer updated should wait for a mailing list discussion?

Reply:
But +1 to saying it is deprecated already, which we've been saying for a while.

Reply:
Done